pkgs/development/python-modules/docformatter/default.nix

```diff
@@ -51,6 +51,12 @@ buildPythonPackage rec {
     pytestCheckHook
   ];
 
+  # Added until the tests are fixed upstream
```

Comment:
The comment would be more useful when it contained why the tests fail in the first place